packages/components/src/local/mapping/index.stories.tsx

```diff
@@ -53,25 +53,25 @@ const OSMTileLayer = {
  * DEFAULT VIEW
  */
 export const Default: React.FC = () => <Mapping
-  tileDiameterMins = {5}
```

Comment:
@foxleigh81  - does @AxelleDRouge need a VS-Code settings file?  Or, should these formatting changes have been caught in the pre-push hook?

Reply:
Yeah, husky should have picked those up before it was pushed. @AxelleDRouge did you run `yarn install` in the repo root? If you only ran it inside the `components` package, you won't have everything you need.

Reply:
I think I did yes.
I'll try again to be sure

Reply:
It worked except for the export node command, but replaced by set for windows it works